README.md

```diff
@@ -12,7 +12,7 @@ cd run/docker
 docker-compose up
 ```
 
-OpenMRS Microfrontends UI is accessible at http://localhost/ui
+New OpenMRS UI is accessible at http://localhost/openmrs/spa
```

Comment:
I am suprised that the call doesn't get intercepted by the Apache ProxyPass directive for `/openmrs`...